### PR TITLE
Preload fade in

### PR DESCRIPTION
**Pull request by denisdani**
For the fallback image I saw this:

https://github.com/PolymerElements/iron-image/pull/44/files/c87d9febfc7a83cc9c75c6228211b0514e13873a#diff-3005a2eaefe34d5d0afb8d08687f8aae

We can maybe listen for the `error-changed` event and decide to change the image with a fallback image.

What do you think @paulvarache ?

### CODE REVIEW

File: kwc-share-cover.html [hunk 1]
```diff
@@ -28,6 +29,12 @@
             }
             .share-cover-image {
                 @apply --kwc-share-cover-image;
+                flex: 1;
+                width: 100%;
+                height: 100%;
+                --iron-image-placeholder: {
+                    background: var(--kwc-share-cover-placeholder, #939ed5);
```
Comment (paulvarache) [inline kwc-share-cover.html]: Set this colour to something like lightgrey by default

Comment (denisdani) [reply kwc-share-cover.html]: 👍

File: kwc-share-cover.html [hunk 2]
```diff
@@ -67,8 +81,23 @@
...
+                fallbackUrl: {
+                    type: String,
+                    value: null,
                 }
-            }
+            },
+            imageError: function(event, par) {
+                let imageError = par.value;
+
+                if(imageError) {
```
Comment (paulvarache) [inline kwc-share-cover.html]: Missing a space here

File: kwc-share-cover.html [hunk 3]
```diff
@@ -67,8 +81,23 @@
...
+                    this.set("imageUrl", this.fallbackUrl);
```
Comment (paulvarache) [inline kwc-share-cover.html]: Use single quotes

Comment (paulvarache) [reply kwc-share-cover.html]: I think It'd be better to use another variable to set the image internally for iron-image. That way we don't change the property `imageUrl` which is set by the user.
Scenarios are:
User set `imageUrl` and `fallbackUrl` => `imageUrl` errors => component sets `imageUrl` to `fallbackUrl` => `imageUrl` is changed
User set `imageUrl` and `fallbackUrl` => `imageUrl` errors => component sets `_imageUrl` to `fallbackUrl` => `imageUrl` is not changed

File: kwc-share-cover.html [hunk 4]
```diff
@@ -66,9 +81,28 @@
...
+                    type: String,
+                    value: null,
                 }
-            }
+            },
+            _imageUrlChanged: function(image) {
+                this.set('_imageUrl', image)
+            },
+            imageError: function(event, par) {
+                let imageError = par.value,
```
Comment (paulvarache) [inline kwc-share-cover.html]: These never change, use const. Also break into multiple line

**Pull request merged**